packages/core/src/__tests__/state-manager.test.ts

```diff
@@ -233,13 +233,13 @@ describe('rewind', () => {
   });
 
   test('return read-only snapshot', async () => {
-    const stream1 = await TileDocument.create<any>(ceramic, INITIAL_CONTENT, { deterministic: true }, { syncTimeoutSeconds: 0 });
-    await anchorUpdate(ceramic, stream1);
+    const metadata = { deterministic: true, family: "family", tags: ["x", "y"], controllers: controllers}
+    const stream1 = await TileDocument.create<any>(ceramic, null, metadata, { syncTimeoutSeconds: 0 });
     await stream1.update({ abc: 321, def: 456, gh: 987 });
     await anchorUpdate(ceramic, stream1);
 
     const ceramic2 = await createCeramic(ipfs, { anchorOnRequest: false });
-    const stream2 = await TileDocument.create(ceramic, INITIAL_CONTENT, { deterministic: true }, { anchor: false, syncTimeoutSeconds: 0 });
+    const stream2 = await TileDocument.create(ceramic, null, metadata, { anchor: false, syncTimeoutSeconds: 0 });
```

Comment:
I kinda feel like all this messing with deterministic documents is distracting for this test given that that isn't what this test is supposed to be about.  What if you just made the stream a normal non-deterministic stream and changed this line to `TileDocument.load(ceramic, stream1.id)`?